projects/packages/forms/src/blocks/contact-form/class-contact-form-block.php

```diff
@@ -166,6 +170,15 @@ public static function load_editor_scripts() {
 				'enqueue'    => true,
 			)
 		);
+
+		$data = array(
+			'defaults' => array(
+				'to'      => wp_get_current_user()->user_email,
+				'subject' => '[' . get_bloginfo( 'name' ) . ']' . ( isset( $post ) ? ' ' . $post->post_title : '' ),
```

Comment:
I'd suggest escaping that data to avoid any potential issues.

Reply:
Good catch!